tests/unit/assessment/clusters/azure-spn-secret-interactive.json

```diff
@@ -0,0 +1,20 @@
+{
+  "autoscale": {
+    "min_workers": 1,
+    "max_workers": 6
+  },
+  "cluster_id": "azure-spn-secret-interactive",
```

Comment:
Can you rename this fixture to tell that this cluster has two service principals configured? Otherwise people would copy-paste it again, because just interactive cluster is very ambiguous